MicrosoftOutlook/microsoft_outlook_modules/client/__init__.py

```diff
@@ -0,0 +1,33 @@
+import requests
+from requests_ratelimiter import LimiterAdapter
+
+from .auth import ApiKeyAuthentication
+from .retry import Retry
+
+
+class ApiClient(requests.Session):
+    def __init__(
+        self,
+        app_id: str,
+        app_secret: str,
+        tenant_id: str,
+        nb_retries: int = 5,
+        ratelimit_per_minute: int = 45,
+    ):
+        super().__init__()
+        self.auth = ApiKeyAuthentication(
```

Comment:
I suggest renaming the authentication class
```suggestion
        self.auth = GraphApiAuthentication(
```